app/client/components/cancel/CancellationReasonReview.tsx

```diff
@@ -432,12 +433,17 @@ const CancellationReasonReview = () => {
 					)
 				) : (
 					<>
-						<h3 id="save_title">
```

Comment:
Is this ID used for anything? I can't find any references to it elsewhere in the codebase. If it turns out it _is_ doing something then the `<Heading>` component will probably need an optional `id` prop.

Reply:
I wonder if it was put in there as a hook for a test. I can't find any other reference to this in the codebase either so I would say it's safe to bin it off 👍